SalesforceCommerceCloud/pwa-kit
# Empty Cart recognizes registered users

**GUS**: (https://gus.my.salesforce.com/a07AH000000SW5GYAW)

## How to test-drive this PR
- Checkout and run locally
- Go to empty cart as guest, observe it gives you the option to sign in
- Sign In - observe the empty cart recognizes you are signed in
